public/app/features/alerting/unified/components/contact-points/DuplicateMessageTemplate.tsx

```diff
@@ -31,11 +31,13 @@ const NewMessageTemplate = ({ match }: Props) => {
     return null;
   }
 
-  return <DuplicateTemplateView
-    alertManagerSourceName={selectedAlertmanager!}
-    config={data}
-    templateName={decodeURIComponent(name)}
-  />;
+  return (
```

Comment:
Just some linting fixes here and in the mock server handler! Changes are all good though :+1:

Reply:
thanks!